src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArray = [];
+  const copyOfState = JSON.parse(JSON.stringify(state));
+
```

Comment:
[Checklist](https://github.com/mate-academy/js_stateful-clones/blob/master/checklist.md) p.1

Reply:
so what? i'am not mutate object :D 🍡

Reply:
ok, up to you. I just noticed that there are two ways in the checklist to do that. I don't know this way, sorry :)

Reply:
Wow... why do u need `JSON parse/stringify`?